src/main/java/org/justinbaur/bankteller/service/Time.java

```diff
@@ -2,18 +2,23 @@
 
 import java.time.LocalDateTime;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class Time {
     private static Time time = null;
 
+    private static final Logger LOG = LoggerFactory.getLogger(JsonFileHandler.class);
```

Comment:
Update to Time class for logger

Reply:
Updated Logger Time class and also found one in tests. Thanks